src/pages/display/index.js

```diff
@@ -445,6 +445,7 @@ export default class Display extends Component {
                         mimeType,
                         uri: uri.split('//')[1],
                         metadata: nft,
+                        pdfscroll: false
```

Comment:
Not a review, just want to understand: 
I dont see the connection between the changes in PdfComponent and this. Is this a pdfComponent too? just confused because I see `src/components/media-types/index.js` , where it is clear that there is a pdfComponent implementation there, but not here. In any case, if this component is being used in other places where it is NOT a pdf, maybe a more genereic name would be better, or a hierarchy with a new component that should be used only for pdf elements would be more cleaner and better to maintain.